pkg/guacanalytics/patchPlanning_test.go

```diff
@@ -194,6 +247,45 @@ func ingestTestData(graphInput string, ctx context.Context, client graphql.Clien
 				logger.Errorf("Error in ingesting isDependency: %v\n", err)
 			}
 		}
+	case "hasSLSA":
```

Comment:
change these to function calls pointing to graph something like `simpleHasSLSAGraph` since this is confusing as it has the same name as the HasSLSA predicate